src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/tests/test_webapp_commands.py

```diff
@@ -0,0 +1,84 @@
+﻿#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from azure.cli.core.utils.vcr_test_base import (ResourceGroupVCRTestBase,
+                                                JMESPathCheck)
+
+#pylint: disable=method-hidden
+#pylint: disable=line-too-long
+#pylint: disable=bad-continuation
```

Comment:
I cloned from other test files :-), but regardless let me remove the unnecessary skips. Likely i will keep the `line-too-long`